.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Chat/Internal/InternalAzureChatDataSourceEndpointVectorizationSource.cs

```diff
@@ -1,8 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Diagnostics.CodeAnalysis;
+
 namespace Azure.AI.OpenAI.Chat;
 
+[Experimental("AOAI001")]
```

Comment:
Since we don't have Export-API here, we should push a manual APIView iteration and double-check that all the right things have Experimental.

Reply:
Yep, for sure. I linked it in the description!